src/main/webapp/content/scss/themes/_dark-variables.scss

```diff
@@ -337,6 +337,13 @@ $quiz-scoring-info-student-modal-question-wrong: $warning;
 $quiz-question-background: $neutral-dark;
 $quiz-question-border-color: $neutral-dark-l-20;
 
+$quiz-visual-question-border-color: #d3d3d3;
+$quiz-visual-answer-correct-container-color: #53a351;
+$quiz-visual-answer-correct-container-hover-color: #90c48e;
+$quiz-visual-answer-wrong-container-hover-color: #f3837a;
+$quiz-visual-answer-add-container-color: #508ac7;
+$quiz-visual-answer-add-container-hover-color: lighten($quiz-visual-answer-add-container-color, 20%);
+
```

Comment:
Please use default colors here. We have themed presets for "success" (answer correct) and "danger" (answer wrong). The add container colors seems to be some sort of blue, please use the "primary" color here.
If you need to use custom colors, please prepare a very good reason and justify this, as documented at https://docs.artemis.cit.tum.de/dev/guidelines/client-design/